### ISSUE TITLE
question about the final cls layer 

### ISSUE THREAD

**Issue by Parsifal133** [vkverma01/EFT#3]
I noticed that the paper mentioned that the parameters of other layers except EFT will not be updated after the first task training. The code you provided does not set the gradient of the last classification layer to zero. Is the last classification layer a global parameter layer or a local parameter layer?

**Comment by Parsifal133** [comment]
In cifar100-10_816_class_incremental.ipynb，you set the [0,7,14,21,28,35,42,49,56,63,70,77,84,91,98,105,112,119,126,133]-th layer's requires_grad=False，the final cls layer is not in this list. Did I miss something? Thanks for your advice!
